esmvalcore/_recipe.py

```diff
@@ -503,6 +511,7 @@ def _fx_list_to_dict(fx_vars):
 
 def _update_fx_settings(settings, variable, config_user):
```

Comment:
Something for another pull request, but what about time dependent fx variables? It looks like `clip_start_end_year` is not applied to those, this will lead to problems when trying to attach them as ancillary variables.